src/program/signal_handlers.py

```diff
@@ -36,32 +34,6 @@ def check_speaker_event_camp_consistency(sender, instance, **kwargs):
                     )
 
 
-def check_speaker_camp_change(sender, instance, **kwargs):
-    if instance.pk:
-        for event in instance.events.all():
-            if event.camp != instance.camp:
-                raise ValidationError(
-                    {
-                        "camp": "You cannot change the camp a speaker belongs to if the speaker is associated with one or more events."
-                    }
-                )
-
-
-def eventinstance_pre_save(sender, instance, **kwargs):
-    """ Save the old instance.when value so we can later determine if it changed """
-    try:
-        # get the old instance from the database, if we have one
-        instance.old_when = sender.objects.get(pk=instance.pk).when
-    except sender.DoesNotExist:
-        # nothing found in the DB with this pk, this is a new eventinstance
-        instance.old_when = instance.when
-
-
-def eventinstance_post_save(sender, instance, created, **kwargs):
-    """ Send an email if this is a new eventinstance, or if the "when" field changed """
-    if created:
-        add_event_scheduled_email(eventinstance=instance, action="scheduled")
-    else:
-        if instance.old_when != instance.when:
-            # date/time for this eventinstance changed, send a rescheduled email
-            add_event_scheduled_email(eventinstance=instance, action="rescheduled")
+def event_session_post_save(sender, instance, created, **kwargs):
+    """ Make sure we have the number of EventSlots we need to have, adjust if not """
+    instance.fixup_event_slots()
```

Comment:
Looking around this rather massive diff, I don't see that the removed email notification is reinstated anywhere. Just asking in case this was a mistake? Should anyone be notified if EventSession is changed, or is it fine?

Reply:
How perceptive, you are correct, they have not been reenabled, they are waiting for #516 (content team needs to come up with the text for them).  Thanks!